src/app/relay/mutations/CreateProjectMediaMutation.js

```diff
@@ -58,8 +58,17 @@ class CreateProjectMediaMutation extends Relay.Mutation {
   }
 
   getFiles() {
-    const { image, video } = this.props;
-    const file = this.props.mode === 'image' ? image : video;
+    const {
+      mode, image, video, audio,
```

Comment:
can we simplify this and merge all these different media type variables into, say, `file` ?

Reply:
Done